sway-lsp/Cargo.toml

```diff
@@ -10,6 +10,8 @@ description = "LSP server for Sway."
 
 [dependencies]
 dashmap = "5.3.4"
+forc = { version = "0.16.1", path = "../forc" }
```

Comment:
Looks like you might not need this anymore now that `check` is in `forc-pkg`?

Reply:
I still need to access `forc::utils::SWAY_GIT_TAG` inside of `sway-lsp` so I can call into the `check` function in `forc-pkg`